### PR TITLE
Drop ubuntu18 and llvm 7,8

### PR DESCRIPTION
**Pull request by tgymnich**

### CODE REVIEW

File: .github/workflows/bcload.yml [hunk 1]
```diff
@@ -39,4 +34,4 @@ jobs:
           cd enzyme/build
           cmake .. -DLLVM_EXTERNAL_LIT=`which lit` -DCMAKE_BUILD_TYPE=${{ matrix.build }} -DLLVM_DIR=/usr/lib/llvm-${{ matrix.llvm }}/lib/cmake/llvm
     - name: make check-bcpass
-      run: cd enzyme/build && make check-bcpass -j`nproc`
+      run: cd enzyme/build && make -j check-bcpass
```
Comment (wsmoses) [inline .github/workflows/bcload.yml:37]: We still want nprocs here

Comment (tgymnich) [reply .github/workflows/bcload.yml:37]: Why?

Comment (wsmoses) [reply .github/workflows/bcload.yml]: Ninja is, make is not sadly

File: enzyme/Enzyme/CMakeLists.txt [hunk 2]
```diff
@@ -44,32 +44,6 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 list(APPEND ENZYME_SRC SCEV/ScalarEvolutionExpander.cpp)
 list(APPEND ENZYME_SRC TypeAnalysis/TypeTree.cpp TypeAnalysis/TypeAnalysis.cpp TypeAnalysis/TypeAnalysisPrinter.cpp TypeAnalysis/RustDebugInfo.cpp)
 
-if (${LLVM_VERSION_MAJOR} LESS 8)
```
Comment (wsmoses) [inline enzyme/Enzyme/CMakeLists.txt:47]: Do we want to add a cmake assertion that LLVM_VERSION_MAJOR >= 9?

**Review by wsmoses — APPROVED** [review]

File: enzyme/Enzyme/CMakeLists.txt (hunk 2 above)
Comment (tgymnich) [reply enzyme/Enzyme/CMakeLists.txt:47]: done

**Review by wsmoses — APPROVED** [review]

**Pull request merged**